src/main/resources/assets/pneumaticcraft/patchouli_books/book/zh_cn/entries/copper_backtank.json

```
@@ -0,0 +1,32 @@
+{
+  "name": "铜背罐",
+  "icon": "create:copper_backtank",
+  "category": "pneumaticcraft:compressed_creativity",
+  "pages": [
+    {
+      "type": "text",
+      "text": "方块形态和物品形态的$(item)铜背罐$()都有了全新功能！$(br)其最多可充至3 bar。$(br2)在方块形态下，其底部会与各种供气端口连接，可用作简易空气压缩机。多余的空气会被直接排出。"
+    },
+    {
+      "type": "image",
+      "images": ["compressedcreativity:textures/gui/bt_block.png"],
+      "border": true,
+      "title": "铜背罐",
+      "text": "铜背罐用作压缩器"
+    },
+    {
+      "type": "text",
+      "text": "在物品形态下，其余$(l:pneumaticcraft:components/air_canister)空气罐$()功能类似。也即为其充气，就和任何其他能装压缩空气的物品一样。"
```

typo 🤣 also minor changes
```suggestion
      "text": "在物品形态下，其与$(l:pneumaticcraft:components/air_canister)空气罐$()功能类似。也即可以充气，就和任何压缩空气容器一样。"
```